coursebuilder/modules/explorer/_static/components/pages/tools-page.html

```diff
@@ -1,497 +1,272 @@
 <link rel="import" href="/static/polymer-1.2.0/polymer/polymer.html">
 
 <dom-module id="tools-page">
-  <template>
-    <div id="tools" class="page-wrapper">
-      <div class="section blue-section">
-        <div class="blue-section-inner">
-          <h2>Tools to help you grow</h2>
-          <p>Whether you want to improve your mobile site or be more visible on the web, consider
-            this your toolkit
-            for digital<br>success.</p>
+    <template>
+      <div id="tools" class="page-wrapper">
+        <div class="section blue-section">
+          <div class="blue-section-inner">
+            <h2>Tools to help you grow</h2>
+            <p>Whether you want to improve your mobile site or be more visible on the web, consider this your toolkit
+              for digital<br>success.</p>
+          </div>
         </div>
-      </div>
-      <!--  Tools  -->
-      <div class="section tools-page">
-        <div class="tools-simple-container tools-first-section">
-          <h2 class="tools-section-label">Online Presence Tools</h2>
-          <div class="row ">
-            <div class="col m12 l6 s12">
-              <div class="tools-card tools-horizontal course-card">
-                <div class="tools-card-image">
-                  <img class="icon-img" src="/modules/explorer/_static/images/wix.jpg">
-                </div>
-                <div class="tools-card-stacked">
-                  <div class="tools-card-content">
-                    <h2>Wix</h2>
-                  </div>
-                  <div class="card-action">
-                    <div class="visit-link">
-                      <a href="#!">VISIT SITE</a>
+        <!--  Tools  -->
+        <div class="hr-sect">Google Tools</div>
+            <div class="row">
+                <div class="col l4 m6 s12">
+                    <div class="card">
+                        <img class="card-img-top" src="/modules/explorer/_static/images/google-search.jpg" alt="Card image cap">
+                        <div class="card-block">
+                            <h4 class="card-title">Google Search</h4>
+                            <p class="card-text">Research with precision. Simple tools and tips to help you get better results, faster.</p>
+                            <a href="#!explorer" class="card-link">VIEW LESSONS</a>
+                            <a href="#!tools" class="card-link">GO TO TOOLS</a>
```

Comment:
@andela-iakande href for each tool points to the tool as appears on https://newslab.withgoogle.com/training/tools hence "GO TO TOOL" as opposed to "TOOLS"

Reply:
@andela-iakande the hrefs should point to the tool please copy them over from  https://newslab.withgoogle.com/training/tools

Reply:
@chege Done